geofront/server.py

```diff
@@ -1063,7 +1124,8 @@ def main_parser() -> argparse.ArgumentParser:  # pragma: no cover
                         help='IP address of a client allowed to override '
                              'url_scheme via the X-Forwarded-Proto header. '
                              'useful when it runs behind reverse proxy. '
-                             '-d/--debug option disables this option')
+                             '-d/--debug option disables this option. '
+                             '(deprecated)')
```

Comment:
If this option has any behavior anymore it should be removed (and recorded to changelog) rather deprecated.